### DISCUSSION TITLE
文件夹和网页链接不能分享

### DISCUSSION THREAD

**Discussion by SalicylicAcid** [localsend/localsend#408] [L-简体中文] [answered]
文件夹没法共享，另外设备之间相互发送网页链接是以文件的形式发送的，我希望在目标设备的浏览器能够直接打开。

**Comment by Kyowapoke** [comment, marked as answer]
1.9.0已实现文件夹共享，但如果可以把接受的文件以文件夹的形式保留，有这么个选项那么更加完美了

**Reply by SalicylicAcid** [reply]
OK，网页链接可以分享了，能否跟随快速保存设置，以便在目标设备直接复制/打开？还有就是如果能加入右键菜单就更完美了，比如右击文件/文件夹分享，以及右击网页分享，非常感谢！

**Reply by abandon-jw3** [reply]
> OK，网页链接可以分享了，可以否跟随快速保存设置，以方便在目标设备直接连接复制/打开？还有就是如果可以加入右键菜单就更完美了，比如右击文件/文件夹分享，以及右击网页分享，非常感谢！

右键分享太重要了！！！！
